.github/workflows/cron-direct-build.yml

```diff
@@ -23,6 +23,11 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version: "1.24"
+
+      - name: Install Pulumi dev release
+        shell: bash
+        run: curl -fsSL https://get.pulumi.com | sh -s -- --version dev
```

Comment:
We should use our own action here.  

I think 

```
- uses: pulumi/actions@v6
  with:
    pulumi-version: dev
```

Should do it.

If you do want to go the curl route, I think we'd have to change the `$PATH` for the next step as well, but `pulumi/actions` is the officially recommended way to install the CLI I think.